heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/YarnLauncher.java

```diff
@@ -161,7 +162,7 @@ Configuration getHMDriverConf() {
         .set(HeronDriverConfiguration.HTTP_PORT, 0)
         .set(HeronDriverConfiguration.VERBOSE, false)
         .set(YarnDriverConfiguration.QUEUE, queue)
-        .set(DriverConfiguration.DRIVER_MEMORY, driverMemory)
+        .set(DriverConfiguration.DRIVER_MEMORY, driverMemory.asMegabytes())
```

Comment:
@mycFelix I now think I am not the right person to review this PR. You can think about [Bill's review](https://github.com/twitter/heron/pull/1728#pullrequestreview-23143032) and wait for @ashvina's comment.